scripts/docker/build.cmd

```diff
@@ -53,7 +53,17 @@ cd %APP_HOME%
     echo ##############################################
     echo.
 
-    dotnet publish WebService      --configuration %CONFIGURATION% --output bin\Docker
+    :: DotNet 3.x:
+    :: If a relative path is specified when publishing a project, the output directory generated 
+    :: is relative to the current working directory, not to the project file location.
+
+    :: DotNet 2.x:
+    :: If a relative path is specified when publishing a project, the output directory generated 
+    :: is relative to the project file location, not to the current working directory.
```

Comment:
Suggestion: this description is very wordy and redundant; consider deduping redundant language to make it more clear and concise -- I had to read it a couple of times to understand the differences between the 2.x and 3.x scenarios.